examples/pyqt5_opencv/app/main.py

```diff
@@ -0,0 +1,44 @@
+import os
+import sys
+from PyQt5.QtCore import pyqtSlot
+from PyQt5.QtCore import QThread
+from PyQt5.QtGui import QImage
+from PyQt5.QtGui import QPixmap
+from PyQt5.QtWidgets import QApplication
+from PyQt5.QtWidgets import QMainWindow
+from PyQt5.uic import loadUi
+
+from camera import CameraDevice
```

Comment:
It made sense for me once I realised that you can never run `.py` files inside a package as scripts. You always launch them with the `-m` flag, like `python -m app.main` (to run app/main.py). Then it's simple: importing any file in the same package is a relative import.